docs/source/_static/proto_doc.html

```diff
@@ -63,57 +63,57 @@
       }
 
       td p:nth-child(1) {
-        text-indent: 0;
+        text-indent: 0;  
       }
 
-
-      .field-table td:nth-child(1) {
+       
```

Comment:
Lot's of extraneous whitespace got added to this file. I assume it's also autogenerated, but if not might be worth stripping it back out.

Reply:
It's definitely autogenerated. I minified it in [652bab0](https://github.com/sandialabs/firewheel/pull/48/commits/652bab0df5868954810a423a151fa24922ab9511), which greatly reduced filesize!